.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,17 +5,22 @@
 
 ### Fixed Issues
 <!---
-Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing.
+1. Please replace GH_LINK with a URL link to the GitHub issue this Pull Request is fixing.
+2. Please replace GH_LINK_ISSUE(COMMENT) with a URL link to your GitHub comment, which contains your proposal (i.e. the proposal that was approved by Expensify).
+
 Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
-It MUST be an entire link to the issue; otherwise, the linking will not work as expected.
+It MUST be an entire link to the github issue and your comment proposal ; otherwise, the linking will not work as expected.
 
 Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):
 
 $ https://github.com/Expensify/App/issues/<number-of-the-issue>
+$ https://github.com/Expensify/App/issues/<number-of-the-issue(comment)>
 
 Do NOT only link the issue number like this: $ #<number-of-the-issue>
 --->
-$ GH_LINK
+$ GH_LINK   
+$ GH_LINK_ISSUECOMMENT
```

Comment:
Ah maybe this should be:
`GH_LINK_ISSUE(COMMENT)` to be consistent with the steps above

Reply:
@michaelhaxhiu I have checked the code and it should be all fine if we not include the dollar sign, there is no reason to have it here and it will not conflict with the matching logic. We might even prepend the proposal word to this 🤷 

```suggestion
Proposal: GH_LINK_ISSUECOMMENT
```

Reply:
Cool - let's just make sure we refer to it correctly in the guidance prescribed above. So like let's use the same terminology `Proposal: GH_LINK_ISSUECOMMENT` above